### PR TITLE
glib: 2.50.0 (Mountain) Lion patch update

### PR DESCRIPTION
**Pull request by mvkorpel**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

This is related to https://github.com/Homebrew/formula-patches/pull/64 .

### CODE REVIEW

**Comment by tschoonj** [comment]
@ilovezfs I really like this failure: though the test itself runs successfully, it outputs its messages on Sierra in German for some reason, and are an exact translation of the expected English ones 😄

**Comment by ilovezfs** [comment]
@tschoonj maybe we can overload the match operator to consider translation matches to be actual matches :)

**Comment by tschoonj** [comment]
😄 

Looks like a locale issue to me. Did someone by mistake set the Sierra bot to German?

**Comment by ilovezfs** [comment]
Yeah, __CF_USER_TEXT_ENCODING is set to 0x1F6:0:2 instead of 0x1F6:0:0 for some reason

**Comment by ilovezfs** [comment]
`println "cat /Users/brew/.CFUserTextEncoding".execute().text` prints 0x2, so there's the culprit.

**Comment by mvkorpel** [comment]
Is there something I need to do to get this accepted? I realize this is a low priority issue because it only affects unsupported OS versions.

File: Formula/glib.rb [hunk 1]
```diff
@@ -63,8 +63,8 @@ class Glib < Formula
   # also applied to configure and gio/Makefile.in
   if MacOS.version < :mavericks
     patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/59e4d32/glib/gnotification-mountain.patch"
-      sha256 "723def732304552ca55ae9f5b568ff3e8a59a14d512af72b6c1f0421f8228a68"
+      url "https://raw.githubusercontent.com/mvkorpel/formula-patches/mountain-glib-2.50.0/glib/gnotification-mountain.patch"
```
Comment (MikeMcQuaid) [inline Formula/glib.rb]: Can you update this URL to Homebrew's now it's merged? Thanks!

**Comment by MikeMcQuaid** [comment]
@BrewTestBot test this please

**Comment by MikeMcQuaid** [comment]
Merging as the weird locale issue isn't on this PR.

**Pull request merged**